### ISSUE TITLE
Blocks: Show a preview of premium blocks

### ISSUE THREAD

**Issue by pento** [Automattic/jetpack#15045]
The fix for #14743 added an upgrade message in the post preview for premium blocks.

In order to give the post author a clearer idea of what the block will look like, we should render a preview version of the block along with the upgrade message.

#### Steps to reproduce the issue

1. Open the editor on a free Simple site.
2. Add a premium block (eg, OpenTable, Calendly, Simple Payments, VideoPress).
3. Preview the post.
4. See that an upgrade nudge is shown.

#### What I expected

I would expect a preview of the block to be displayed. In the case of blocks like VideoPress, which require an upgrade before videos can be uploaded, it could use the block `example` data as a placeholder.

**Comment by jeherve** [comment]
Closing as duplicate of #14881
